### ISSUE TITLE
Use a button element for close button?

### ISSUE THREAD

**Issue by i-like-robots** [Financial-Times/o-tooltip#39]
Currently the tooltip close button is an anchor element with `role=button` and a non-existant fragment ID:

```js
// Build and append the close button
const button = document.createElement('a');
button.className = 'o-tooltip-close';
button.setAttribute('role', 'button');
button.setAttribute('href', '#void');
button.setAttribute('aria-label', 'Close');
button.setAttribute('title', 'Close');
this.tooltipEl.appendChild(button);
```

This is currently driving one my Pa11y tests nuts. Is there a strong reason for this _not_ to be a button element?

**Comment by notlee** [comment]
🤔 I can't think of one!

**Comment by i-like-robots** [comment]
Fixed by #40
